meta-xt-domx/recipes-core/dbus/dbus-cxx_0.10.0.bb

```diff
@@ -6,7 +6,7 @@ DEPENDS = "boost dbus libsigc++-2.0"
 
 LIC_FILES_CHKSUM = "file://COPYING;md5=d32239bcb673463ab874e80d47fae504"
 
-SRC_URI = "git://github.com/dbus-cxx/${PN}.git;nobranch=1;tag=${PV}"
+SRC_URI = "https://github.com/dbus-cxx/${PN}.git;nobranch=1;tag=${PV}"
```

Comment:
As I see, `nobranch=1` is already present. Why mention it in the commit message?

Reply:
fixed